.eslintrc.js

```diff
@@ -33,6 +33,8 @@ module.exports = {
             ],
             rules: {
                 "@typescript-eslint/no-non-null-assertion": "off",
+                "simple-import-sort/imports": "off",
+                "simple-import-sort/exports": "off",
```

Comment:
This should not be necessary with the steps mentioned here: https://github.com/ankitects/anki/pull/2395#issuecomment-1445777657

Reply:
Thanks!